source/postprocess/dynamic_topography.cc

```diff
@@ -110,9 +117,14 @@ namespace aspect
               // for each of the quadrature points, evaluate the
               // stress and compute the component in direction of the
               // gravity vector
+              
+              double dynamic_topography_x_volume = 0;
+              double volume = 0;
+              Point<dim> location;
+
               for (unsigned int q=0; q<quadrature_formula.size(); ++q)
                 {
-                  const Point<dim> location = fe_values.quadrature_point(q);
+                  location = fe_values.quadrature_point(q);
```

Comment:
Or that, but I think that's actually a nice touch -- the code is not wrong if one used a higher order quadrature formula with the exception of the re-use of `location` further down below.